scripts/travis_gdal_install.sh

```diff
@@ -44,7 +44,7 @@ GDALOPTS="  --with-ogr \
             --without-php \
             --without-ruby \
             --without-python \
-            --with-static-proj4=/usr/lib"
+            --with-proj"
```

Comment:
@snowman2 since the proper option depends on the GDAL version, let's remove all forms from the literal GDALOPTS and then conditionally append the proper option depending on the GDAL version.